Item/Item.js

```diff
@@ -133,36 +320,40 @@ const ItemBase = kind({
 /**
  * Agate specific behaviors to apply to [Item]{@link agate/Item.ItemBase}.
  *
+ * @class ItemDecorator
  * @hoc
  * @memberof agate/Item
- * @mixes ui/Slottable.Slottable
  * @mixes ui/Item.ItemDecorator
- * @mixes spotlight/Spottable.Spottable
- * @mixes ui/Marquee.MarqueeController
+ * @mixes agate/Marquee.MarqueeController
+ * @mixes ui/Slottable.Slottable
  * @mixes agate/Skinnable.Skinnable
+ * @mixes spotlight/Spottable.Spottable
  * @public
  */
 const ItemDecorator = compose(
-	ForwardRef({prop: 'componentRef'}),
-	Slottable({slots: ['label', 'slotAfter', 'slotBefore']}),
-	Pure,
+	MarqueeController({marqueeOnFocus: true, invalidateProps: ['inline']}),
 	UiItemDecorator,
-	Spottable,
-	MarqueeController({marqueeOnFocus: true}),
-	Skinnable
+	Slottable({slots: ['label', 'slotAfter', 'slotBefore']}),
+	Skinnable,
+	Spottable
```

Comment:
The order is different from sandstone's. Is there a particular reason for this?

Reply:
The reason was to order these alphabetically. I can change it back to match Sandstone.